docs/extra-docs.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
This needs to be in `docs/docsite/`, not in `docs/`. (The `docsite/` is also missing in the next file's path.)